xds/internal/client/filter_chain.go

```diff
@@ -108,7 +117,11 @@ type FilterChainManager struct {
 
 // destPrefixEntry is the value type of the map indexed on destination prefixes.
 type destPrefixEntry struct {
-	net *net.IPNet // The actual destination prefix.
+	// The actual destination prefix. Set to nil for unspecified prefixes.
+	net *net.IPNet
+	// Set to true for unspecified prefixes.
+	absent bool
```

Comment:
Do we need both of these?  What happens if `net` is `nil` and `absent` is `false`?  What if `net` is non-`nil` and `absent` is `true`?  We should not have both pieces of state if those are illegal combinations.